### PR TITLE
Use wabt spec module

### LINKED ISSUE #2: Use wabt-rs wast2json?

### PR DESCRIPTION
**Pull request by pepyakin**
Resolves #2

### CODE REVIEW

**Comment by pepyakin** [comment]
I guess, I like the former external iteration approach more than this one...

File: spec/src/run.rs [hunk 1]
```diff
@@ -359,216 +323,162 @@ fn run_action(
...
+		match result {
+			Ok(result) => {
+				let spec_expected = expected.iter().cloned().map(spec_to_runtime_value).collect::<Vec<_>>();
+				let actual_result = result.into_iter().collect::<Vec<RuntimeValue>>();
+				for (actual_result, spec_expected) in
+					actual_result.iter().zip(spec_expected.iter())
+				{
+					assert_eq!(actual_result.value_type(), spec_expected.value_type());
+					// f32::NAN != f32::NAN
+					match spec_expected {
```
Comment (NikVolf) [inline spec/src/run.rs]: `match *spec_expected { .. }` ?

**Comment by NikVolf** [comment]
Looks cool and easy to add similar thing to parity-wasm

**Comment by pepyakin** [comment]
Hey @NikVolf what do you think on this one https://github.com/pepyakin/wabt-rs/issues/8 ?

**Comment by pepyakin** [comment]
WIP until changes from https://github.com/pepyakin/wabt-rs/issues/8 are integrated

File: spec/src/run.rs [hunk 2]
```diff
@@ -520,55 +425,34 @@ pub fn spec(name: &str) {
...
-            } |
-            &test::Command::AssertMalformed {
-                line, ref filename, ..
-            } |
-            &test::Command::AssertUnlinkable {
-                line, ref filename, ..
-            } => {
-                let module_load = try_load(tmpdir.as_ref(), filename, &mut spec_driver);
+            CommandKind::AssertInvalid { module, .. }
+            | CommandKind::AssertMalformed { module, .. }
```
Comment (NikVolf) [inline spec/src/run.rs]: isn't `|` better at the end of the lines?

Comment (pepyakin) [reply spec/src/run.rs]: I think this depends on the taste. 
As this is an automatic formatting I rather leave it as is.

**Pull request merged**